DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Indigo\Bundle\CookieConsentBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('indigo_cookie_consent');
+
+        $rootNode
+            ->children()
+                ->arrayNode('options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('message')->defaultValue('message')->end()
+                        ->scalarNode('dismiss')->defaultValue('dismiss')->end()
+                        ->scalarNode('learnMore')->defaultValue('learn_more')->end()
+                        ->scalarNode('link')->end()
+                        ->scalarNode('container')->end()
+                        ->scalarNode('theme')->end()
```

Comment:
i would add some `info()` to these configurations - its not self-evident what container or theme are. maybe also just drop them, if they are for the template. the user can overwrite the twig template in app/Resources/IndigophpCookieConsentBundle/twig/ if he needs to change the layout.